contrib/gce/cloud-init/master.yaml

```diff
@@ -0,0 +1,199 @@
+#cloud-config
+
+users:
+- name: etcd
+  homedir: /var/etcd
+  lock_passwd: true
+  ssh_redirect_user: true
+
+write_files:
+# Setup containerd.
+  - path: /etc/systemd/system/containerd-installation.service
+    permissions: 0644
+    owner: root
+    content: |
+      # installed by cloud-init
+      [Unit]
+      Description=Download and install containerd binaries and configurations.
+      After=network-online.target
+
+      [Service]
+      Type=oneshot
+      RemainAfterExit=yes
+      ExecStartPre=/bin/mkdir -p /home/containerd
+      ExecStartPre=/bin/mount --bind /home/containerd /home/containerd
+      ExecStartPre=/bin/mount -o remount,exec /home/containerd
+      ExecStartPre=/usr/bin/curl --fail --retry 5 --retry-delay 3 --silent --show-error -H "X-Google-Metadata-Request: True" -o /home/containerd/configure.sh http://metadata.google.internal/computeMetadata/v1/instance/attributes/containerd-configure-sh
+      ExecStartPre=/bin/chmod 544 /home/containerd/configure.sh
+      ExecStart=/home/containerd/configure.sh
+
+      [Install]
+      WantedBy=containerd.target
+
+  - path: /etc/systemd/system/containerd.service
+    permissions: 0644
+    owner: root
+    content: |
+      # installed by cloud-init
+      [Unit]
+      Description=containerd container runtime
+      Documentation=https://containerd.io
+      After=containerd-installation.service
+
+      [Service]
+      Restart=always
+      RestartSec=5
```

Comment:
Should there be a `type=notify` there? For other clouds except for GCE I see that original `containerd.service` from repo-root has been `install`ed. For GCE, the change for notify type service is kept out. This might cause some systemd services dependent on containerd to start & fail before container runtime is up.

Reply:
This version was migrated over from CRI repository. It was created and managed by GCE folks, so I'll leave it to them if they want to change it going forward.